p2panda-rs/src/document/document_view_id.rs

```diff
@@ -40,6 +41,23 @@ impl DocumentViewId {
     pub fn graph_tips(&self) -> &[Hash] {
         self.0.as_slice()
     }
+
+    /// Returns a hash over the graph tips constituting this view id.
+    pub fn hash(&self) -> Hash {
+        let graph_tip_bytes = self
+            .0
+            .clone()
+            .into_iter()
+            .flat_map(|graph_tip| graph_tip.to_bytes())
+            .collect();
+        Hash::new_from_bytes(graph_tip_bytes).unwrap()
+    }
+}
+
+impl Display for DocumentViewId {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.hash().as_str())
+    }
 }
```

Comment:
This is interesting. I had imagined that the way we want to identify `SchemaId`s would still be by their array of hashes (for debugging purposes) but that the hash string would be a useful identifier in some specific cases. With this thinking i'd expect `Display` to show the hash list. But I'm open to thinking differently about this!

Reply:
I agree, the `DocumentViewId` should be displayed as .. a document view id! I would suspect the hash living in something like `DocumentViewHash`

Reply:
Alright, I'll add a string representation for document view ids and use that to implement `Display` in `Schema` and `SchemaId`